DataManagement/BranchTutorial.Rmd

```diff
@@ -32,7 +32,7 @@ knitr::include_graphics("Pics/git.jpg")
 <br/><br/>
 
 ### Fork repo (on github)
-Got to Github, login and go to the repository you want to fork. Click on the fork button on the top right of the page. A new window will appear and you can give the forked repository a new name.
+Go to Github, login and go to the repository you want to fork. Click on the fork button on the top right of the page. A new window will appear and you can give the forked repository a new name.
 
 ```{r fork repo, echo=FALSE, out.width = '100%'}
```

Comment:
you can set the default for echo in the setup chunk (line 12) to FALSE, then you don't need to give it every time